deeplinkdispatch-base/src/main/java/com/airbnb/deeplinkdispatch/DeepLinkResult.java

```diff
@@ -7,11 +7,14 @@
   private final boolean successful;
   private final String error;
   @Nullable private final String uriString;
+  private final String componentName;
 
-  public DeepLinkResult(boolean successful, @Nullable String uriString, String error) {
+  public DeepLinkResult(boolean successful, @Nullable String uriString, String error,
```

Comment:
would be nice to have a comment explaining what the format of componentName is

Reply:
What do you mean by format? Do you mean something beyond https://github.com/airbnb/DeepLinkDispatch/pull/260/files/256268ae06907ec3dcb220ace44280a99cc7e624#diff-0616d20e785ed562a4ad36be4f682e4dR35

Reply:
yes, "name" is vague. There is simple name, qualified name, canonical name, etc. It's pretty safe to assume that FQN name is what is meant here, but it should be explicit

Reply:
Okay. Do you think it makes sense to duplicate the comment on [L36](https://github.com/airbnb/DeepLinkDispatch/pull/260/files#diff-0616d20e785ed562a4ad36be4f682e4dR36) here or to delete that one and move it here?

Reply:
I decided to duplicate it across both the property and the getter^